targets/lenovo-x1/everything.nix

```diff
@@ -149,7 +151,7 @@
 
               # UI applications
               profiles = {
-                applications.enable = false;
+                applications.enable = true;
```

Comment:
dont enable that here, it has side effects at the moment :)

Reply:
accident, removed it